<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImplicitSkinSettings</class>
 <widget class="QGroupBox" name="ImplicitSkinSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Implicit Skin Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="contactAngle">
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>55.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="sigma">
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.350000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Iterations</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Skinining</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="implicitSkin">
        <property name="text">
         <string>Implicit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="lbwSkin">
        <property name="text">
         <string>LBW</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sigma</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="renderMesh">
     <property name="text">
      <string>Render Mesh</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="loadAnimation">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="iterations">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="animationFileText">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="browseAnimationFiles">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Contact Angle</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="wireframe">
     <property name="text">
      <string>Wireframe</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="isoSurface">
     <property name="text">
      <string>Iso-Surface</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
